test/helpers/database.reset.ts

```diff
@@ -0,0 +1,11 @@
+import dbClient from '@/config/database';
+
+const resetDatabase = async () => {
+  return await dbClient.$transaction([
+    dbClient.searchResult.deleteMany(),
+    dbClient.account.deleteMany(),
+    dbClient.user.deleteMany(),
```

Comment:
Another way is to use the [cascade delete](https://www.prisma.io/docs/concepts/components/prisma-schema/relations/referential-actions). But this way works too ✋ 

```suggestion
    dbClient.user.deleteMany(),
```